genref/README.md

```diff
@@ -0,0 +1,89 @@
+# Config API generator
+
+This is a Config API reference generator that uses the
+[k8s.io/gengo](https://godoc.org/k8s.io/gengo) project to parse types and
+generate API documentation from it.
+
+This tool relies only on the Go source code containing the type definitions.
+For references to external type definitions, especially those from the
+`k8s.io/kubernetes` repo, you can tell the tool to generate links to the
+existing docs, which can be the GoDocs if no better option exists.
+
+## Try it out
+
+1. Clone this repository.
+
+2. Make sure you have go1.15+ installed. Run `make` to build the `genref`
+   binary:
+
+   ```
+   cd genref
+   make
+   ```
+
+3. Generate reference doc for some API types.
+
+   ```shell
+   ./genref -include kubelet-config --output output/html 
+   ```
+
+4. Visit `output/html/kubelet-config.v1beta1.html` to view the results.
+
+## Customization
+
+### Use a different config file
+
+You can use the `-c` flag to specify a different configuration file for
+generating the config API references. For example,
+
+```shell
+./genref -c myconfig.yaml
+```
+
+### Specify the package
+
+You can modify the `config.yaml` file to customize the generated API
+reference output.
+You can also use `-include` and `-exclude` flags for the `genref` binary
+to customize which package to include or exclude respectively.
+The value for the `-include` and `-exclude` flags must be one of the
+`name` of the `apis` listed in the `config.yaml` file, for example,
+`kubelet-config`, `kube-scheduler-config`.
```

Comment:
Can you use the `include` flag with multiple component configs:
-include kubelet-config, kube-scheduler-config